stacks/api/passport/passport.module.ts

```diff
@@ -29,7 +29,7 @@ class PassportCoreModule {
       module: PassportCoreModule,
       imports: [
         CorePassportModule.register({
-          defaultStrategy: options.defaultStrategy || "jwt",
```

Comment:
You should provide the default strategy. When the user does not send a valid schema, we have to authorize the user against this strategy.